src/app/core-ui/main-header/main-header.component.html

```diff
@@ -16,7 +16,7 @@
       mat-ripple
       routerLink="/active/tasks"
     >
-      {{activeWorkContext?.title}}
+      {{T.PDS.TODAY|translate}}
```

Comment:
This way we can allow users to edit the title:
```suggestion
      {{activeWorkContext?.title ==='Today' ? T.PDS.TODAY|translate: activeWorkContext?.title}}
```